lib/parse.js

```diff
@@ -68,7 +68,7 @@ module.exports = (input) => {
     device = _.chain(device)
       .split('\n')
       .filter((line) => {
-        return /^(\s\s-\s)?[a-z]+:/g.test(line);
+        return /^(\s\s-\s)?[a-zA-Z]+:/g.test(line);
```

Comment:
Good catch! 

> Also, why is this testing for \s\s-\s, when none of the example input has that prefix?
Is that something that one of the scripts produces?

Mountpoints have that prefix. For example:

```yaml
device: /dev/disk1
displayName: /dev/disk1
description: "Macintosh HD - APPLE SSD SM0256G"
size: 140999917568
mountpoints:
  - path: "/"
raw: /dev/rdisk1
protected: False
system: True
```